examples/illustrations/Oresteia/Oresteia.json

```diff
@@ -656,8 +660,15 @@
                     ],
                     "uco-observable:keypadUnlockCode": "123789",
                     "uco-observable:IMEI": "359305065690067",
-                    "uco-observable:MSISDN": "[clytemnestra-mobileaccount-uuid]",
-                    "uco-observable:clockSetting": "2017-06-21T06:36:24.35Z",
+                    "uco-observable:MSISDN": [
+                        {
+                            "@id": "kb:clytemnestra-mobileaccount-uuid"
+                        }
+                    ],
```

Comment:
The above is in a json array but I'm not sure if in this case this is OK as I have noted a usage of this that is the similar from the accounts example (https://github.com/casework/CASE-Examples/blob/b0d6daf461b63c6e23d7439ea41050ca60017dd3/examples/illustrations/accounts/accounts.json#L36-L40) 

Also this is what is outlined as the usage in the slides I am working off.